Lib/test/test_typing.py

```diff
@@ -4868,6 +4868,28 @@ def test_valid_uses(self):
         self.assertEqual(C4.__args__, (Concatenate[int, T, P], T))
         self.assertEqual(C4.__parameters__, (T, P))
 
+    def test_invalid_uses(self):
```

Comment:
```suggestion
    def test_invalid_usage(self):
```

Reply:
There's `def test_valid_uses(self):`, so I followed this naming convention 🙂